singer_sdk/testing/templates.py

```diff
@@ -242,7 +243,10 @@ def non_null_attribute_values(self) -> list[t.Any]:
             for r in self.stream_records
             if r.get(self.attribute_name) is not None
         ]
-        if not values:
+        if (
+            not values
+            and self.stream.name not in self.config.ignore_no_records_for_streams
+        ):
```

Comment:
Should we add `self.config.ignore_no_records` to this condition, like in https://github.com/meltano/sdk/blob/6f325721fa72dc464b560dc6c4389babcaca5141/singer_sdk/testing/tap_tests.py#L106-L109?

Reply:
Hmm yeah, that makes sense